package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "description": "commit to GitHub using the API",
-  "main": "dist/index.js",
+  "main": "lib/main.js",
```

Comment:
What is this value used for?

I assumed this was the value the action would run and would therefore be the dist/index.js which is committed to the repo and in this PR.

Reply:
I assumed something similar. Turns out this is [a module reference](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#main), and `dist/index.js` is a standalone file that is built with this `lib/main.js` as the entrypoint.

Reply:
Here's where we reference `dist/index.js` in the action:

https://github.com/getsentry/action-github-commit/blob/eeed373a06dab4762f73177d607078a5f7b7a48c/action.yml#L18